src/Ledger/Transaction.lagda

```diff
@@ -172,13 +173,30 @@ the transaction body are:
   txinsScript : ℙ TxIn → UTxO → ℙ TxIn
   txinsScript txins utxo = txins ∩ dom (proj₁ (scriptOuts utxo))
 
-  lookupScriptHash : ScriptHash → Tx → Maybe Script
-  lookupScriptHash sh tx =
+  refScripts : Tx → UTxO → ℙ Script
+  refScripts tx utxo = goal -- TODO: implement when we do Babbage
+    where
+    -- The Babbage pdf spec says this should be
```

Comment:
Yeah, it's not that anything is mistaken, it's just a design change. You can remove these comments since we don't want them in the PDF.